src/translations/Bahasa_Indonesia.js

```diff
@@ -0,0 +1,91 @@
+/*

```

Comment:
How to convert CRLF enlines to LF (by recommendation order, don't try 3 if you can do 1) : 
1. [Using dos2unix](https://stackoverflow.com/a/27814403)
2. [Using notepad++ *usually the same with every text editors, like atom*](https://stackoverflow.com/a/54245311)
3. [Using powershell](https://stackoverflow.com/a/32064596)

Reply:
I did it by using notepad++. Thank you for the information.